modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -242,126 +241,72 @@ dbms.cluster.discovery.resolver_type=LIST
 +
 [source,cypher]
 ----
 CALL dbms.cluster.showParallelDiscoveryState();
 ----
 +
-The expected result should display `v2ServerCount` as 3.
-`stateComparison` is not expected to match at this stage because the original servers are not visible to the V2 discovery service.
+Now the returned mode for this server must be `V1_OVER_V2` and the `stateComparison` must show that the states are matching.
+If they are not, wait and try again till matching.
 +
 .Expected result
 [queryresult]
 ----
- +---------------------------------------------------------------------------------------------------------+
- | mode         | stateComparison                                          | v1ServerCount | v2ServerCount |
- +---------------------------------------------------------------------------------------------------------+
- | "V1_OVER_V2" | "States are not matching after PT55M36.693S: (score:29)" | "6"           | "3"           |
- +---------------------------------------------------------------------------------------------------------+
++----------------------------------------------------------------+
+| mode         | stateComparison | v1ServerCount | v2ServerCount |
++----------------------------------------------------------------+
+| "V1_OVER_V2" | "Matching"      | "3"           | "3"           |
++----------------------------------------------------------------+
 ----
+. Run the once again the following procedure to turn on `V1` in the background for all servers, but keep `V2` running in the foreground.
 +
-. Deallocate, drop, and shut down servers 1, 2, 3. 
-+
-. Start up servers 7, 8, 9, this time with the setting `dbms.cluster.discovery.version=V2_OVER_V1`.
-+
-image:discovery-service/v2_over_v1.png[]
-+
-The discovery addresses in the settings should include addresses of the new members, and the previous members.
-+
-As an example, for those using the list resolver, the settings for the new servers should include:
-+
-[source, parameters]
+[source,cypher]
 ----
-dbms.cluster.discovery.resolver_type=LIST
-dbms.cluster.discovery.version=V2_OVER_V1
-
-dbms.cluster.discovery.endpoints=server04.example.com:5000,server05.example.com:5000,server06.example.com:5000,server07.example.com:5000,server08.example.com:5000,server09.example.com:5000
-dbms.cluster.discovery.v2.endpoints=server04.example.com:6000,server05.example.com:6000,server06.example.com:6000,server07.example.com:6000,server08.example.com:6000,server09.example.com:6000
+CALL internal.dbms.cluster.moveToNextDiscoveryVersion();
 ----
-+
-. Using `bolt://`, connect to the system database of servers 7, 8, 9 and run the following procedure:
+. Check that transition to `V2_OVER_V1` was successful:
 +
 [source,cypher]
 ----
 CALL dbms.cluster.showParallelDiscoveryState();
 ----
 +
-The output should display `Matching` in the `stateComparison` column.
+Now the returned mode for this server must be `V2_OVER_V1` and the `stateComparison` must show that the states are matching.
```

Comment:
same here - 'the returned mode for this server...' or it should be - 'for the cluster...'?

Reply:
https://github.com/neo4j/docs-operations/pull/1861#discussion_r1809060567